src/netex_idf/stops.rs

```diff
@@ -254,17 +256,16 @@ fn load_stop_points<'a>(
 }
 
 fn load_stops(
-    elem: &Element,
+    frames: &HashMap<FrameType, Vec<&Element>>,
```

Comment:
you might create a new type for `HashMap<FrameType, Vec<&Element>>` 
it will be used a lot

Reply:
Good idea, I'll take a look for another PR.